kafka/sink/samples/kafka-sink-deployment.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
would it make sense to have this thing being reflected by a CRD ?

Reply:
Yes in future